update_service/index.adoc

```diff
@@ -0,0 +1,48 @@
+:context: update-service
+[id="update-service"]
+= Understanding the {product-title} Update Service
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+[id="update-service-install"]
+== Installing the {product-title} Update Service
+include::modules/common-attributes.adoc[]
+
+// You can install the {product-title} Update Service by deploying the Update Service operator, using the {product-title} xref FIXME ../../update_service/install/web-console.adoc[web console] or xref FIXME ../../update_service/install/cli.adoc[CLI].
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+include::modules/update-service-install-web-console.adoc[leveloffset=+2]
+
+include::modules/update-service-install-cli.adoc[leveloffset=+2]
+
+== Additional resources
+
+* For more information on installing Operators, see xref:../operators/admin/olm-adding-operators-to-cluster.adoc#olm-installing-operators-from-operatorhub_olm-adding-operators-to-a-cluster[Installing Operators from the OperatorHub].
+
+include::modules/update-service-graph-data.adoc[leveloffset=+2]
+
+[id="update-service-create-service"]
+== Creating an Update Service
+include::modules/common-attributes.adoc[]
+
+// You can create Update Services using the {product-title} xref FIXME ../../update_service/create-service/web-console.adoc[web console] or xref FIXME ../update_service/create-service/cli.adoc[CLI].
+
+include::modules/update-service-create-service-web-console.adoc[leveloffset=+2]
+
+include::modules/update-service-create-service-cli.adoc[leveloffset=+2]
+
+FIXME-deleting-a-service?
```

Comment:
Neither of the next two items are part of the installing story, so they'd need separate assemblies.

Reply:
this is an `update_service/index.adoc` assembly.  Does it have to be scoped to installation?  I was expecting to handle the whole thing with a single assembly, now that I've pivoted to "modules for everything".

Reply:
This would be a beautiful book, but we build out the books in the topic map instead of at the assembly level. Each separate user story is supposed to be a separate assembly - you're ideally supposed to tie each assembly to a separate user objective and be able to follow the instructions to a single end.